app/helpers/carts_helper.rb

```diff
@@ -1,2 +1,22 @@
 module CartsHelper
+
+  def set_order_summery
+    set_cart
+    @sub_total = 0
+    @tax =  0
+    @shipping_charge = 0
```

Comment:
shipping_charge that means

Reply:
this variable indicates that shipping cost would be free or not